packages/web3torrent/src/constants.ts

```diff
@@ -3,13 +3,19 @@ import {ChannelState} from './clients/payment-channel-client';
 import {bigNumberify} from 'ethers/utils';
 
 export const WEI_PER_BYTE = bigNumberify(1); // cost per byte
-export const BUFFER_REFILL_RATE = bigNumberify(2e4); // number of bytes the leecher wishes to increase the buffer by
+export const BLOCK_LENGTH = 1 << 14; // Standard request length.
+export const PEER_TRUST = 1; //amount of trust between peers. It's equivalent to the amount of request to pre-pay.
+// The recomended value is 5 ( the size of the queue of requests made by the leecher to the seeder)
```

Comment:
Perhaps we should set to the recommended value?

Reply:
It seems to me that `PEER_TRUST` should be proportional to an amount of money, rather than an amount of requests. My concern is that with different sized files, we have different sized pieces, and for the same `PEER_TRUST` we might be pre-paying more or less money?

Reply:
It isn't, although the size of the pieces can vary, the maximum size of a request is constant. It's the BLOCK_LENGTH size. That's what I was explaining on my comment about chunks and pieces.

This allows us to not having any variance in payment once the "uneven" parts change, using a simple variable. For example, usually, the queue is made up of 5 requests to each wire. Setting the PEER_TRUST to 5 is, to me, the optimum size, as it would optimize the communication flow between the peers.